web-client/src/modules/personalData/components/PersonalDataForm/PersonalDataForm.tsx

```diff
@@ -497,14 +519,21 @@ const PersonalDataForm: React.FC<PersonalDataFormProps> = ({
                   : setTermsAndPrivacyAccepted(null)
               }
             >
-              {t('user_data_form.terms_conditions_text')}
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href="https://github.com/reach4help/reach4help/blob/master/CODE_OF_CONDUCT.md"
+              <Trans
+                i18nKey="modules.personal-data.components.PersonalDataForm.terms_conditions_text"
+                values={{
+                  linkText: t(
+                    'modules.personal-data.components.PersonalDataForm.terms_conditions_link_text',
+                  ),
+                }}
               >
-                {t('user_data_form.terms_conditions_link')}
-              </a>
+                {/* eslint-disable-next-line jsx-a11y/anchor-has-content,jsx-a11y/control-has-associated-label */}
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://github.com/reach4help/reach4help/blob/master/CODE_OF_CONDUCT.md"
```

Comment:
We're still referring to the CODE_OF_CONDUCT here, what is the actual link of our terms&conditions page? We can fix it too within this PR.

Reply:
https://github.com/mcverter/reach4help/blob/feature/add-coding-conventions/web-client/docs/CodingConventions/CODING_CONVENTIONS.md

for now only on my branch @btuerker

Reply:
we don't have this ready yet. so ignore this for the time being